docs/standard/datetime/toc.yml

```diff
@@ -39,3 +39,5 @@
     href: resolve-ambiguous-times.md
   - name: "How to: Let users resolve ambiguous times"
     href: let-users-resolve-ambiguous-times.md
+  - name: "DateTime and DateTimeOffset support in System.Text.Json"
+    href: datetime-support-in-system-text-json.md
```

Comment:
file names shouldn't have unnecessary small words, such as a, or, and, the, and in. (https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files)

And also shouldn't repeat a word that is already part of the path. In this case, datetime would be repeated twice in the path.

Reply:
> file names shouldn't have unnecessary small words, such as a, or, and, the, and in. 

Interesting. I would be curious to know the reasoning for this guideline?

I can't find the repo/file for this (gives me 404), but the url seems to use "in":
https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master

Reply:
It's all for SEO, which wouldn't matter for this internal article you linked to. Let me see if I can find some better guidance.

Reply:
I've pushed a change addressing this @mairaw.